yarn.lock

```diff
@@ -17616,10 +17616,10 @@ karma@^6.3.0:
     ua-parser-js "^0.7.30"
     yargs "^16.1.1"
 
-keyborg@^1.1.0:
-  version "1.1.0"
-  resolved "https://registry.yarnpkg.com/keyborg/-/keyborg-1.1.0.tgz#f6886a90b4ec1f4c5272326213110f6d9e831d0e"
-  integrity sha512-AmEH/if+1TG1MkDT4UOAG8ZKTTO2IwJBXq20crsLD3QhMsRivL6OpxZSTBrnbGAsn/QeH/+YIgDCXcNvLSe/Yw==
+keyborg@^1.1.0, keyborg@^1.1.1:
```

Comment:
shouldnt this be migrated together with `tabster` to have same version aligment ?

Reply:
you're right, I updated tabste too in https://github.com/microsoft/fluentui/pull/24376/commits/a9f763511c4f0fc61c21155d90ad2f4adc807d57